libs/sdk-ui-tests/tests/api-regression/geo/__snapshots__/geoPushpin.test.tsx.snap

```diff
@@ -191,6 +191,7 @@ Object {
       "deauthenticate": [Function],
       "isAuthenticated": [Function],
       "onHostname": [Function],
+      "organization": [Function],
```

Comment:
i know this is not your doing, however this snapshot uncovered that the geo chart is not cleaning 'public' props correctly before sending to inner component. it does not remove the `backend` instance which may come on public API.

can you please clean it up while here? the problem is in: libs/sdk-ui-geo/src/GeoPushpinChart.tsx  L78.

we do this for normal charts. for an inspiration how to do it in type-safe way check out: libs/sdk-ui-charts/src/charts/_commons/chartDefinition.ts L12 & L73

should be a quick fix (if you go with it, please add separate commit)

Reply:
Fixed as separate commit as requested.